packages/deck-scripts/helpers/rollup-plugin-angularjs-template-loader.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const path = require('path');
+const MagicString = require('magic-string');
+
+// This rollup plugin finds AngularJS templates and loads them into the $templateCache
+// This is a rollup replacement for the ngtemplate-loader webpack loader
+module.exports = function angularJsTemplateLoader(options = {}) {
+  function isSourceMapEnabled() {
+    return options.sourceMap !== false && options.sourcemap !== false;
+  }
+
+  return {
+    transform(originalCode, id) {
+      let code = originalCode;
+      const templateRegex = /require\(['"]([^'"]+\.html)['"]\)/g;
```

Comment:
I hope this only catches angular templates. Not really sure if pattern will catch some other library usage.

Reply:
We might be able to tighten up the Regex a bit to look for `templateUrl: require(` and any other permutations that we find in angularjs codee

Reply:
we might be able to tighten this regex up to something like `templateUrl\w*:\w*require(` and add any other patterns besides `templateUrl` that is used in the angularjs code.  Should be pretty easy to find this list by grepping the codebase for `/require\(['"]([^'"]+\.html)['"]\)/g` and filtering out all the known patterns.

```
grep -R  "require.....regex_in_grep_format" deck/app/scripts | grep -v "templateUrl: require"
```

Reply:
> Like is it possible to have a .html require for anything other than an angular template? Definitely wouldn't happen within deck, but I'm wondering if some library package may have that statement for something else (may be web components?) and we may end up breaking that?

Yeah that's my concern.  We could probably mitigate in the rollup plugin by checking the name of the source file and skipping if `node_modules` is in the name